Resfeber/Model/EventItems.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  EventItems.swift
+//  Resfeber
+//
+//  Created by Joshua Rutkowski on 12/2/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import MapKit
+
+class EventItem {
+    /// The event's category
+    var category: Category
+    /// The event's name
+    var name: String
+    /// The event's latitude and longitude
+    var location: CLLocation?
+    /// The event's description
+    var description: String?
+    /// The event's start date
+    var startDate: Date?
+    /// The event's end date
+    var endDate: Date?
+    /// The event's list of places to go and things to do.
+    var list: [ListItem]?
+    /// The event's notes
+    var notes: String?
+    
+    /// Default categories for events
+    enum Category: String {
+        case accommodation = "Accommodation"
+        case activity = "Activity"
+        case excursion = "Excursion"
+        case restaurant = "Restaurant"
+        case event = "Event"
+        case other = "Other"
+    }
+    
+    init(category: Category, name: String) {
```

Comment:
It might be worth giving category a default value of `.other`
```suggestion
    init(name: String, category: Category = .other) {
```